recipes-graphics/mesa/egl-gbm_1.1.0.bb

```diff
@@ -0,0 +1,30 @@
+DESCRIPTION = "EGL backend for libgbm"
+HOMEPAGE = "https://github.com/NVIDIA/egl-gbm"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://COPYING;md5=6288a8dacdfbb0c5d8e4cf6dade6d203"
+DEPENDS = "eglexternalplatform libdrm virtual/libgbm"
+
+SRC_REPO = "github.com/NVIDIA/egl-gbm.git;protocol=https"
+SRCBRANCH = "main"
+SRC_URI = "git://${SRC_REPO};branch=${SRCBRANCH}"
+# 1.1.0 tag
+SRCREV = "39932b2cc4f44cdadd553cc931f3bebd4e348d10"
+
+SRC_URI += "\
+    file://0001-Tegra-workarounds.patch \
+    file://nvidia_gbm.json \
+"
+
+REQUIRED_DISTRO_FEATURES = "opengl"
+
+S = "${WORKDIR}/git"
+
+inherit meson pkgconfig features_check
+
+do_install:append() {
+    install -d ${D}${datadir}/egl/egl_external_platform.d
+    install -m 0644 ${WORKDIR}/nvidia_gbm.json ${D}${datadir}/egl/egl_external_platform.d/15_nvidia_gbm.json
+}
+
+FILES:${PN} += "${datadir}/egl"
+INSANE_SKIP:${PN} = "dev-so"
```

Comment:
Do we need an `RDEPENDS` on `tegra-udrm-gbm` in this recipe, to make sure your wrapper gets pulled in?  This might not be the right place for it, but we do need something to pull in that package so users don't have to do it themselves.

Reply:
I've added this as an RDEPENDS on mesa's libgbm, because that's the component that really needs the backend support.